src/Integration/Notifications/NotificationIndicator.xaml

```diff
@@ -0,0 +1,95 @@
+﻿<UserControl x:Name="UserControl" x:Class="SonarLint.VisualStudio.Integration.Notifications.NotificationIndicator"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:local="clr-namespace:SonarLint.VisualStudio.Integration.Notifications"
+        xmlns:integration="clr-namespace:SonarLint.VisualStudio.Integration.WPF">
+    <UserControl.Resources>
+        <integration:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" />
+        <integration:BoolToVisibilityConverter x:Key="InvertedBoolToVisibilityConverter" TrueValue="Collapsed"
+                FalseValue="Visible" />
+        <ControlTemplate x:Key="ImageButtonTemplate" TargetType="ToggleButton">
+            <Grid x:Name="Root" Width="30" Height="16" Background="Transparent">
+                <Grid Margin="3,1,1,1" HorizontalAlignment="Left">
+                    <Path Data="M13.9,16h-1.8v-0.5c0-6.2-5.1-11.3-11.5-11.3H0.1V2.3h0.5c7.3,0,13.3,5.9,13.3,13.1V16z"
+                            Fill="#D5D5D5" />
+                    <Path
+                            Data="M13.6,11.3l-0.1-0.5c-0.9-3.9-4.1-7.2-8.1-8.3L4.9,2.3l0.5-1.6l0.5,0.1c4.6,1.3,8.2,5.1,9.2,9.5l0.1,0.5L13.6,11.3z"
```

Comment:
Please either change the following 2 paths or the previous one to use only one formatting.